src/App.tsx

```diff
@@ -1,14 +1,28 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo, Filter } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [handleClose, setHandleClose] = useState<boolean>(false);
```

Comment:
You should be able to deal without that state. Try to conditionally render `TodoModal` basing on different condition.